src/utils.ts

```diff
@@ -27,6 +27,15 @@ function sanitizeFileName(s: string) {
 	return s.replace(illegalChars, '-');
 }
 
+/**
+ * Create a new markdown note and populate with the location
+ * @param app The obsidian app instance
+ * @param newNoteType The location format to encode as
+ * @param directory The directory path to put the file in
+ * @param fileName The name of the file
+ * @param location The coordinate
```

Comment:
location, not coordinate

Reply:
This was copied over from my other pull request so I must have missed that.
Can I suggest we rename the location arguments to coordinates?
In some places it is a bit ambiguous if location is a point in the world or a point in a file.
I suggest "coordinate" for a world point and "location" for a file point. I will change this in a future pull request if you agree.